.github/workflows/validate-flusight-testhub-config.yaml

```diff
@@ -0,0 +1,80 @@
+name: Hub Config Validation (R)
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: main
+    paths:
+      - 'inst/testhubs/flusight/hub-config/**'
+      - '!inst/testhubs/flusight/**README**'
```

Comment:
At first I thought this was trying to ignore the README, but in bold, but now I realize what the regex is doing and it's clever!